diff --git a/setup.py b/setup.py
index 63d4264ce2976feb04d57e5729304e236210b388..ca08d0245cd5ab5ed11072aef9e88119b1e26ba6 100644
--- a/setup.py
+++ b/setup.py
@@ -31,7 +31,7 @@ try:
     # If Cython is installed, transpile the optimized Cython module to C and compile as a .pyd to be distributed
     from Cython.Build import cythonize
     print("Cython is installed, building creedsolo module")
-    extensions = cythonize([ Extension('creedsolo', ['creedsolo.pyx']) ])
+    extensions = cythonize([ Extension('creedsolo', ['creedsolo.pyx']) ], force=True)
 except ImportError:
     # Else Cython is not installed (or user explicitly wanted to skip)
     if '--native-compile' in sys.argv:
